.github/workflows/build.yml

```diff
@@ -125,36 +129,20 @@ jobs:
       # Possibly related: actions/checkout#290
       run: git fetch origin --force --tags
 
-    - name: "[macOS] Set up cmake"
-      uses: jwlawson/actions-setup-cmake@v1.9
-      # Ubuntu 20.04 should use the CMake version from the repos, and Visual
-      # Studio on Windows comes with its own CMake version anyway.
-      if: runner.os == 'macOS'
-      with:
-        # This should always match the minimum required version in
-        # our CMakeLists.txt
-        cmake-version: '3.16.x'
-
```

Comment:
This change is unrelated to this PR.
If we remove it we have no check if our minimum CMake dependent still applies.
If you like to remove it, let's do it separately.

Reply:
It is already not used for Windows nor Ubuntu. If we do decide to keep it for some OS, macOS is not a good choice because macOS 11 requires a newer version of CMake.

Reply:
At least no topic of this PR.